README.md

```diff
@@ -119,18 +154,18 @@ kind: Pod
 metadata:
   name: aws-cli
   labels:
-	name: aws-cli
+    name: aws-cli
```

Comment:
I guess I should be adding a `.editorconfig` to avoid this kind of spacing issue ;) No biggies, no need to change anything.